org.eclipse.xtend.core/src/org/eclipse/xtend/core/compiler/XtendCompiler.java

```diff
@@ -375,7 +374,7 @@ private void prependLocalTypesIfFieldInitializer(XExpression obj, ITreeAppendabl
 		}
 	}
 	
-	public void _toJavaStatement(RichString richString, ITreeAppendable b, boolean isReferenced) {
+	private void _toJavaStatement(RichString richString, ITreeAppendable b, boolean isReferenced) {
```

Comment:
I'd prefer this to remain at least protected since I'm aware of languages that inherit from Xtend.

Reply:
OK, I'll update that. I thought Xtend was not meant to be API ;)

Reply:
True, yet no reason to do unwarranted changes that can be easily avoided.

Reply:
OK, I reverted to protected methods here https://github.com/eclipse/xtext/pull/2890/commits/7ac3f496fd906b5cdbeef5932ff0d17388d405b6